src/components/Header/Header.js

```diff
@@ -1,13 +1,16 @@
 import React from 'react'
+import onClickOutside from 'react-onclickoutside'
```

Comment:
This component was already installed in package.json. We can use it in other places too if we need. Very useful

https://www.npmjs.com/package/react-onclickoutside